app_dart/test/src/service/fake_buildbucket.dart

```diff
@@ -22,7 +22,7 @@ class FakeBuildBucketClient extends BuildBucketClient {
 
   Future<Build>? scheduleBuildResponse;
   Future<SearchBuildsResponse>? searchBuildsResponse;
-  Future<BatchResponse>? batchResponse;
+  Future<BatchResponse> Function()? batchResponse;
```

Comment:
one test was asserting that a closure was being invoked; however, that closure was actually being invoked before it was set here. Change this type to a callback so that it gets invoked in place doing the actual `batch()` call.